### PR TITLE
some first updates to README to bring it up to date with high-level.md

### LINKED ISSUE #77: README needs improvements
terminology: "app store" needs to change to something else

also it is confusing the way the manifiest fields required by the store is presented to the webxdc developers (as a Rust struct)

we probably need a clear separation in the documentation for: bot admins / deployment and for app developers

### PR DESCRIPTION
**Pull request by hpk42**
closes #77 
addresses  #44 in that we don't specify any author anymore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,54 @@
...
+
+TODO: make bot available via sending a message to: xdcstore@testrun.org 
+
+TODO: insert QR code here for setting up contact with the xdcstore bot
+
+After establishing initial contact the bot will send a "xdc store" app which provides
+the central point of interaction for users: 
+
+- scroll the app index list and hit the "share/sendToChat/forward" button on any app 
+  of your likening
```
Comment (adbenitez) [inline README.md:25]:
```suggestion
- scroll the app index list and hit the "share/sendToChat/forward" button on any app 
  of your liking
```

Comment (adbenitez) [inline README.md:25]:
```suggestion
  of your liking
```

File: PUBLISHING.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+3. Once all requirements are met, and someone in the review chat confirms the submission
+   the app will become available in the app index the store xdc makes available to users. 
+
+### App Publishing Requirements
+
+Here we record the metadata for each application submission that we intend 
+to keep about each submitted app version, noted down in Rust-struct style: 
+
+```rust
+Copy code
```
Comment (adbenitez) [inline PUBLISHING.md:25]: is this a to-do line that was forgotten to be removed? :thinking:

**Comment by Septias** [comment]
I generally like the style of uppercasing the first letter (In lists etc. and also for Consecutive Words in a Longer Technical Term) instead of the mostly lowercase style you are using atm, but other than that it's already a great improvement :+1: 
This should only give you a general idea of my opinion, but as you are writing the whole file, you can totally go for your own style :)

**Comment by hpk42** [comment]
did some more changes, and adapted the capitalizing style.  

please review and approve.

File: README.md [hunk 3]
```diff
@@ -49,22 +70,61 @@ with files `appstore.xdc`, `review-helpr.xdc` and `submit-helper.xdc`.
...
+
+Notes: 
+
+- `app_id` and `version` fields are used by the bot to sort submitted apps so that 
+  each app occurs with the latest version. 
+
+- `app_id` MUST be alphanumberic && case-insensitive and only "." is
+  allowed as special character 
+
+- `version` MUST adhere to https://semver.org/
```
Comment (Septias) [inline README.md:124]: Punktuation

File: README.md [hunk 4]
```diff
@@ -1,31 +1,52 @@
-## Appstore Bot for DeltaChat
-Welcome to the DeltaChat Appstore Bot. You can start using it today by contacting it at this `<email>`. The appstore bot acts both as distribution platform as well as publishing service.
+## Webxdc app store
```
[the remark this answers is not in the document]
Comment (hpk42) [reply README.md]: i updated this -- i think we should talk of "store xdc" and "store bot" and avoid "app store".

**Review by Simon-Laux — APPROVED** [review]
lgtm

File: README.md [hunk 5]
```diff
@@ -49,22 +71,67 @@ with files `appstore.xdc`, `review-helpr.xdc` and `submit-helper.xdc`.
...
+    cp example-xdcs/*.xdc import/
+    cargo run -- import
+```
+
+### Per-App metadata 
+
+The store bot uses the following meta data for each xdc app,
+noted down in rust-struct style for now: 
+
 ```
```
Comment (Simon-Laux) [inline README.md:99]: maybe add `rust` here for a colourful code block

File: README.md [hunk 6]
```diff
@@ -1,31 +1,54 @@
-## Appstore Bot for DeltaChat
-Welcome to the DeltaChat Appstore Bot. You can start using it today by contacting it at this `<email>`. The appstore bot acts both as distribution platform as well as publishing service.
+## Webxdc store: discover and share the latest apps to your chats
```
Comment (adbenitez) [inline README.md:1]: what about "WebXDC" ? it is about time we settle in an style for the term ;)

File: README.md [hunk 7]
```diff
@@ -49,22 +72,67 @@ with files `appstore.xdc`, `review-helpr.xdc` and `submit-helper.xdc`.
...
+    cp example-xdcs/*.xdc import/
+    cargo run -- import
+```
+
+### Per-App metadata 
+
+The store bot uses the following meta data for each xdc app,
+noted down in rust-struct style for now: 
+
 ```
```
Comment (adbenitez) [inline README.md:100]:
```suggestion
```rust
```

File: README.md [hunk 8]
```diff
@@ -49,22 +72,67 @@ with files `appstore.xdc`, `review-helpr.xdc` and `submit-helper.xdc`.
...
+    pub image: String,                   // Taken from .xdc file
+    pub source_code_url: Option<String>, // Taken from manifest.toml
+
+    // Fields not specified yet but required for store bot purposes 
+    pub description: String,             // Taken from manifest.toml
+    pub app_id: String,                  // Taken from manifest.toml 
+    pub version: String,                 // Taken from manifest.toml 
+
+    // Derived data (not coming from manifest.toml or .xdc file) 
+    pub submitter_uri: Option<String>,   // determined by bot during interaction
```
Comment (adbenitez) [inline README.md:112]:
```suggestion
    pub submitter_uri: Option<String>,   // determined by bot during interaction, "unknown" for imported apps
```

**Review by Septias — APPROVED** [review]

**Pull request merged**